index.html

```diff
@@ -190,7 +190,7 @@ <h1>Contact me</h1>
       <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
     </div>
     <form action="post">
-      <input type="text" value="Ndonyi" placeholder="Your Name Here" id="input" required/>
+      <input type="text" value="Ndonyi" placeholder="Your Name Here" max-lenght="30" id="input" required/>
       <input type="email" value="mauricendonyi40@gmail.com" placeholder="Your Email Here" id="input" required/>
       <textarea id="textpanel" cols="30" rows="9" placeholder="Write Your Message Here"></textarea> 
```

Comment:
- [ ] Kindly add an  attribute ` maxlength="500"` to your `<textarea>`  for validation reasons